packages/xml-views-completion/test/providers/attributeName/prop-event-assoc-spec.ts

```diff
@@ -37,12 +37,23 @@ const radioButtonGroupProperties = [
   "valueState",
   "width"
 ];
+
+const radioButtonGroupSpecialProperties = [
+  "id",
+  "models",
+  "bindingContexts",
+  "objectBindings",
+  "metadataContexts",
+  "Type"
+];
+
 const radioButtonGroupEvents = ["select"];
 const uiCoreControlAssociations = ["ariaDescribedBy", "ariaLabelledBy"];
 const allPropsEventsAssociations = uiCoreControlProperties
   .concat(uiCoreControlEvents)
   .concat(uiCoreControlAssociations)
   .concat(radioButtonGroupProperties)
+  .concat(radioButtonGroupSpecialProperties)
```

Comment:
it would be nice to have a separated test that only checks these specials properties.
That will give useful indication if this feature every breaks.